internal/build/imgsrc/dockerfile_builder.go

```diff
@@ -221,7 +221,7 @@ func (*dockerfileBuilder) Run(ctx context.Context, dockerFactory *dockerClientFa
 
 	build.SetBuilderMetaPart2(buildkitEnabled, serverInfo.ServerVersion, fmt.Sprintf("%s/%s/%s", serverInfo.OSType, serverInfo.Architecture, serverInfo.OSVersion))
 	if buildkitEnabled {
-		imageID, err = runBuildKitBuild(ctx, docker, opts, relDockerfile, buildArgs)
+		imageID, err = runBuildKitBuild(ctx, docker, opts, dockerfile, buildArgs)
```

Comment:
do we need this fix on [line 234](https://github.com/superfly/flyctl/pull/2849/files#diff-c76cea6ab9d205c8b3838eecbf83e23ea3793acde3f112ca0f421dabd120b29eR234) as well?

Reply:
Seems non-Buildkit build is fine. https://flyio.discourse.team/t/specifying-a-non-pwd-path-to-dockerfile-in-fly-toml-spits-out-an-error/3975 cc @Roadmaster